packages/types/src/aztec/l2Contract.ts

```diff
@@ -118,7 +118,7 @@ export type ChicmozL2UtilityFunctionBroadcastedEvent = z.infer<
 >;
 
 export const CONTRACT_STANDARDS = {
-  "0.0.0-73e84dcc": ["token", "dripper"],
+  "0.0.0-83476cda": ["token", "dripper"],
```

Comment:
I think we should be able to support both?

```suggestion
  "0.0.0-83476cda": ["token", "dripper"],
  "0.0.0-73e84dcc": ["token", "dripper"],
```